### PR TITLE
run Unit tests with Julia 1.6 until 1.9 and nightly build

### PR DESCRIPTION
**Pull request by SimeonEhrig**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -25,6 +24,44 @@ unit_tests_julia1.9:
...
+  extends: .untit_test_template
+  parallel:
+    matrix:
+      - JULIA_VERSION: ["1.6", "1.7", "1.8", "1.9"]
+  image: julia:$JULIA_VERSION
+
+unit_tests_nightly:
+  extends: .untit_test_template
+  # use the same baseimage like the official julia images
+  image: debian:bookworm-slim
```
Comment (szabo137) [inline .gitlab-ci.yml:37]: Is this the base image, which is also used for `julia:$JULIA_VERSION` above?

**Review by szabo137 — COMMENTED** [review]
Most of the PR is fine, there is just one smaller question.

File: .gitlab-ci.yml (hunk 1 above)
Comment (SimeonEhrig) [reply .gitlab-ci.yml:37]: Yes, see here: https://hub.docker.com/layers/library/julia/1.9.3/images/sha256-44b6dfeb1646a41b5de39294740ba603b6d18164e6077c5e2bb5b64677b41f74?context=explore

Comment (szabo137) [reply .gitlab-ci.yml:37]: Nice! Thanks for clarifying.

**Review by szabo137 — APPROVED** [review]
No additional questions, or concerns. Merging!

**Pull request merged**